### PR TITLE
Changes For Start_date

### PR DESCRIPTION
**Pull request by srivastava-jyoti**
Targets #{316}

Checklist:

- [x] Corrected > in start_date.
- [x] To disable previous dates.

### CODE REVIEW

File: resources/views/hr/job/create.blade.php [hunk 1]
```diff
@@ -96,3 +96,21 @@ class="form-control"
...
+<script>
+    var date = new Date();
+    var tdate = date.getDate();
+    if(tdate < 10){
+        tdate = '0' + tdate;
+    }
+    var month = date.getMonth() + 1;
+    if(month < 10){
+        month = '0' + month;
+    }
```
Comment (GauravGusain98) [inline resources/views/hr/job/create.blade.php:110]:
```suggestion
    if (month < 10) {
        month = '0' + month;
    }
```

File: resources/views/hr/job/create.blade.php [hunk 2]
```diff
@@ -96,3 +96,21 @@ class="form-control"
...
 </div>
 @endsection
+
+@section('js_scripts')
+<script>
+    var date = new Date();
+    var tdate = date.getDate();
+    if(tdate < 10){
+        tdate = '0' + tdate;
+    }
```
Comment (GauravGusain98) [inline resources/views/hr/job/create.blade.php:106]:
```suggestion
    if (tdate < 10) {
        tdate = '0' + tdate;
    }
```

File: resources/views/hr/job/create.blade.php [hunk 3]
```diff
@@ -96,3 +96,21 @@ class="form-control"
...
+    var month = date.getMonth() + 1;
+    if(month < 10){
+        month = '0' + month;
+    }
+    var year = date.getUTCFullYear();
+    var minDate = year + "-" + month + "-" + tdate;
+    document.getElementById("start_date").setAttribute('min',minDate);
+    document.getElementById("end_date").setAttribute('min',minDate);
```
Comment (GauravGusain98) [inline resources/views/hr/job/create.blade.php:114]:
```suggestion
    document.getElementById("start_date").setAttribute('min', minDate);
    document.getElementById("end_date").setAttribute('min', minDate);
```

File: resources/views/hr/job/edit.blade.php [hunk 4]
```diff
@@ -138,3 +138,22 @@ class="form-control"
...
+    var month = date.getMonth() + 1;
+    if(month < 10){
+        month = '0' + month;
+    }
+    var year = date.getUTCFullYear();
+    var minDate = year + "-" + month + "-" + tdate;
+    document.getElementById("start_date").setAttribute('min',minDate);
+    document.getElementById("end_date").setAttribute('min',minDate);
+</script>
+@endsection
```
Comment (GauravGusain98) [inline resources/views/hr/job/edit.blade.php:158]: Please write the scripts in app.js file. You should not write the JS in the blade files.  Correct it at other places as well.

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Please check the comments and make the requested changes.

File: resources/views/hr/job/edit.blade.php (hunk 4 above)
Comment (srivastava-jyoti) [reply resources/views/hr/job/edit.blade.php:158]: @GauravGusain98  I will correct the required changes.

File: app.js [hunk 5]
```diff
@@ -0,0 +1,1086 @@
+/**
```
Comment (GauravGusain98) [inline app.js:1]: You should not create a new file for JS. You can write your code into the `Modules/HR/Resources/assets/js/app.js` file

File: resources/views/hr/job/create.blade.php [hunk 6]
```diff
@@ -96,3 +96,4 @@ class="form-control"
     </form>
 </div>
 @endsection
+
```
Comment (GauravGusain98) [inline resources/views/hr/job/create.blade.php:99]: Remove this as there are no changes in this file

File: app.js (hunk 5 above)
Comment (GauravGusain98) [reply app.js:1]: Also try to test it once in your local machine after making the changes

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Please check the comments

File: app.js (hunk 5 above)
Comment (srivastava-jyoti) [reply app.js:1]: @GauravGusain98 I have not created new JS file. I have added code from lines 1005-1017 in app.js  file and from line 902 hr module code starts. Also, it is working fine on my local.
Screenshot (56)

Comment (srivastava-jyoti) [reply app.js:1]: @GauravGusain98 I was facing the same issue. I have run this command **npm run watch** and it's working fine on my local.
I have taken reference from here:[link](https://stackoverflow.com/questions/53425186/laravel-reactjs-changes-are-not-reflecting-after-change-in-my-jsreactjs-file)

**Pull request closed without merging**